docs/setup/new_environment.md

```diff
@@ -0,0 +1,162 @@
+# Setting up a new CommCareHQ environment
+
+This document will walk you through the process of setting up a new monolith server to run CommCareHQ
+
+## Pre-requesites
+
+* A server with Ubuntu 18.04 installed (referred to as the "monolith" from here)
+* Access to the monolith via SSH with a user who has sudo access
+
+## Step 1: Install Required System Packages
+
+Install pip on the monolith
+
+``` bash
+$ sudo apt install python-pip
+$ sudo pip install ansible virtualenv virtualenvwrapper --ignore-installed six
+```
+
+## Step 2: Download and configure the commcare-cloud environment
+
+1. Clone the sample environment into the `environments` folder:
+ ``` bash
+$ git clone https://github.com/emord/supreme-octo-potato.git environments
+ ```
+
+1. Encrypt the provided ansible vault file by running:
+
+ ``` bash 
+$ ansible-vault encrypt ~/environments/monolith/vault.yml
+ ```
+
+ Enter a strong password when prompted, and save this password somewhere safe as you will need it for any future changes to this file, as well as when you deploy and configuration changes to this machine.
+
+### Add passwords to the vault file
+
+In the vault file, change all fields that have the value 'CHANGE ME' to strong, different, passwords
+
+``` bash
+$ ansible-vault edit ~/environments/monolith/vault.yml`
+```
+
+### Set the network interface name
+
+1. Find the name of the network interface of your machine, and note it down. You can do this by running 
+
+ ``` bash
+ $ ip addr
+ ```
+ This will give an output that looks like:
+
+ ```
+ $ ip addr
+ 1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
+     link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+     inet 127.0.0.1/8 scope host lo
+        valid_lft forever preferred_lft forever
+     inet6 ::1/128 scope host 
+        valid_lft forever preferred_lft forever
+ 2: enp0s3: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel state UP group default qlen 1000
+     link/ether 08:00:27:48:f5:64 brd ff:ff:ff:ff:ff:ff
+     inet 10.0.2.15/24 brd 10.0.2.255 scope global dynamic enp0s3
+        valid_lft 85228sec preferred_lft 85228sec
+     inet6 fe80::a00:27ff:fe48:f564/64 scope link 
+        valid_lft forever preferred_lft forever
+ ```
+ 
+ Here, the network interface we are interested in is **enp0s3**. Note this value, or copy it to the system clipboard.
```

Comment:
nit: this line (and I believe all lines below `1. Find the name of the network...`) should be indented three spaces to be properly indented and grouped with the first item in the list.

Reply:
Thanks. Github behaves differently to my markdown previewer (`vmd`), so this is helpful to know.